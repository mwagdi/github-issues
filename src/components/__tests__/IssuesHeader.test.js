import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import IssuesHeader from '../IssuesHeader';

test('IssuesHeader renders correctly', () => {
  const wrapper = shallow(<IssuesHeader />);
  expect(toJSON(wrapper)).toMatchSnapshot();
})
