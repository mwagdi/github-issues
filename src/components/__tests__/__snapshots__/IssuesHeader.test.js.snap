// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IssuesHeader renders correctly 1`] = `
<div
  className="issues__header offwhite"
>
  <div
    className="row"
  >
    <div
      className="issues__heading flex-container align-center"
    >
      <svg
        aria-hidden="true"
        className="octicon octicon-repo"
        fill="#959da5"
        height="16"
        version="1.1"
        viewBox="0 0 12 16"
        width="12"
      >
        <path
          d="M4 9H3V8h1v1zm0-3H3v1h1V6zm0-2H3v1h1V4zm0-2H3v1h1V2zm8-1v12c0 .55-.45 1-1 1H6v2l-1.5-1.5L3 16v-2H1c-.55 0-1-.45-1-1V1c0-.55.45-1 1-1h10c.55 0 1 .45 1 1zm-1 10H1v2h2v-1h3v1h5v-2zm0-10H2v9h9V1z"
          fillRule="evenodd"
        />
      </svg>
      <span
        className="issues__header_link blue"
      >
        facebook
      </span>
      <span>
        /
      </span>
      <span
        className="issues__header_link blue bold"
      >
        react
      </span>
    </div>
    <div
      className="issues__navbar flex-container"
    >
      <a
        className="issues__navbar_link text-center"
      >
        Code
      </a>
      <Link
        className="issues__navbar_link issues__navbar_link--active text-center"
        replace={false}
        to="/"
      >
        Issues
      </Link>
      <a
        className="issues__navbar_link text-center"
      >
        Pull requests
      </a>
      <a
        className="issues__navbar_link text-center"
      >
        Projects
      </a>
      <a
        className="issues__navbar_link text-center"
      >
        Wiki
      </a>
      <a
        className="issues__navbar_link text-center"
      >
        Insights
      </a>
    </div>
  </div>
</div>
`;
